extension-e2e/src/page-objects/pages/importing-existing-wallet.page.ts

```diff
@@ -0,0 +1,12 @@
+import { createPageElement } from '../../utils/search.utils';
+import { Page } from '../page.class';
+
+export class ImportExistingWalletPage extends Page {
+  seedPhraseInput = createPageElement('ImportWalletFromSeedPhraseSelectors.SeedPhraseInput');
+  nextButton = createPageElement('ImportWalletFromSeedPhraseSelectors.NextButton');
+
```

Comment:
could you reuse enums instead of string values here? (You did it right for `WelcomePage`)